Backpack-SwiftUI/Nudger/Classes/BPKNudger.swift

```diff
@@ -45,14 +46,17 @@ public struct BPKNudger: View {
     ///     Must be greater than `min`.
     ///   - step: The step value of the `BPKNudger`.
     ///     Defaults to `1`.
-    public init(value: Binding<Int>, min: Int, max: Int, step: Int = 1) {
+    ///   - accessibilityPrefix: The string which would be added to accessibility identifiers of  `BPKNudger` components.
+    ///     Defaults to `nil`.
+    public init(value: Binding<Int>, min: Int, max: Int, step: Int = 1, accessibilityPrefix: String? = nil) {
```

Comment:
I would solve this using a modifier instead of adding it to the initializers.
There's a big chance we might miss this in future updates and a modifier would make it more robust api wise.